android/engine/src/main/java/org/smartregister/fhircore/engine/util/extension/TaskExtension.kt

```diff
@@ -29,6 +29,19 @@ fun Task.hasStarted() =
     this.executionPeriod.hasStart() &&
```

Comment:
Maybe for this we can re-use the new method `Task.executionStartIsBeforeOrToday`